libexec/alluxio-config.sh

```diff
@@ -120,3 +128,4 @@ ALLUXIO_WORKER_JAVA_OPTS+=" -Dalluxio.logger.type=${ALLUXIO_WORKER_LOGGER:-WORKE
 # Client specific parameters based on ALLUXIO_JAVA_OPTS.
 ALLUXIO_USER_JAVA_OPTS+=${ALLUXIO_JAVA_OPTS}
 ALLUXIO_USER_JAVA_OPTS+=" -Dalluxio.logger.type=USER_LOGGER"
+ALLUXIO_USER_JAVA_OPTS+=" -Dalluxio.master.audit.logger.type=Null"
```

Comment:
If we do not add -Dalluxio.master.audit.logger.type, then it will be "" (empty string). This will confuse log4j when it configures the loggers using log4j.properties. It will look for the properties for an appender whose name is "". Since it cannot find one, it will complain "Cannot find value for key log4j.appender."".